attestation-service/protos/attestation.proto

```diff
@@ -10,13 +10,53 @@ enum Tee {
     Sample = 4;
     AzSnpVtpm = 5;
     CSV = 6;
+    CCA = 7;
 }
 
 message AttestationRequest {
+    // TEE enum. Specify the evidence type
     Tee tee = 1;
-    string nonce = 2;
-    string evidence = 3;
+
+    // Base64 encoded evidence. The alphabet is URL_SAFE_NO_PAD.
+    // defined in https://datatracker.ietf.org/doc/html/rfc4648#section-5
+    string evidence = 2;
+
+    // Base64 encoded runtime data slices. Every single string will
+    // be base64 decoded and all the result slices will then be concentrated
+    // to get a compound one. This compound one will then be accuulated into
+    // a digest which is used as the expected runtime data to check against
+    // the one inside evidence.
+    //
+    // The alphabet is URL_SAFE_NO_PAD.
+    // defined in https://datatracker.ietf.org/doc/html/rfc4648#section-5
+    repeated string runtime_data = 3;
```

Comment:
this seems partly redundant because the api already has `evidence` which is according to the KBS protocol spec ("The KBS does not parse or analyze the attestation evidence, it forwards it to the Attestation-Service for verification."). The part that is missing is `nonce` so it could be a dedicated entry here (instead of `runtime_data`).

Reply:
Sorry,I realized I mixed things so my earlier comment can be ignored.